packages/location-management/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@opensrp/location-management",
+  "version": "0.0.0",
+  "main": "dist/index.js",
+  "author": "OpenSRP Engineering",
+  "license": "Apache-2.0",
```

Comment:
lets add  the configuration that points to the directory from which types will be resolved [i.e](https://github.com/onaio/js-tools/blob/a6433bb02a1f5127a44986a532c8a1435f8a0a91/packages/Loaders/package.json#L4)